tools/coreml/test/test_mxnet_converter.py

```diff
@@ -61,35 +61,42 @@ def _get_mxnet_module(net, data_shapes, mode, label_names, input_names=None):
 
 class SingleLayerTest(unittest.TestCase):
     """
-    Unit test class for testing where converter is able to convert individual layers or not.
-    In order to do so, it converts model and generates preds on both CoreML and MXNet and check they are the same.
+    Unit test class for testing where converter is able to convert individual
```

Comment:
The current line width limit should be 100 characters https://lists.apache.org/thread.html/bfbbf8ddd7aa9ce5285735f574107bd120b3d1ad2aea47e38df08dca@%3Cdev.mxnet.apache.org%3E